shared/constants/config.js

```diff
@@ -8,6 +8,7 @@ export const defaultPrivatePrefix = '/private/'
 export const defaultPublicPrefix = '/public/'
 
 // Actions
+export const daemonError = 'config:daemonError'
```

Comment:
i added this but also no other actions/reducer/ flow stuff exists so in my mind this is something that needs to be cleaned up eventually across the whole file